es_utils.py

```diff
@@ -267,3 +267,26 @@ def es_send_external_stats(stats_dict_file_path, opts_dict_file_path, cpu_normal
   sdata["@timestamp"]=file_stamp*1000
   try:send_payload(es_index_name+"-"+week, es_doc_name, index_sha, json.dumps(sdata))
   except Exception as e: print(e.message)
+
+def getExternalsESstats(cmsdist='*', arch='*', lastNdays=30, page_size=0):
```

Comment:
@mrodozov , may be its better to not use cmsdist. There will be no data for new release cycles. How about we pass (optional) a dict of package name and realversions e.g `{"root": "6.22.06", "geant4": "10.6", ....}`. I think before starting the build we can get the list of packages which will be built and then only get the stats of those packages

Reply:
arch only works too and sorting can be done later over names (or even cmsdist branch if matching since it's in the result)